### PR TITLE
Place the table of content feature in the header.

### PR DESCRIPTION
**Pull request by mtias**
Making some tweaks to the placement of the table of contents button.

image

### CODE REVIEW

File: editor/table-of-contents/index.js [hunk 1]
```diff
@@ -27,12 +27,11 @@ function TableOfContents( { blocks } ) {
...
 			renderToggle={ ( { onToggle } ) => (
-				<button
-					className="table-of-contents__toggle"
+				<IconButton
 					onClick={ onToggle }
-				>
-					<Dashicon icon="info" />
-				</button>
+					icon="admin-page"
+					label={ __( 'Information' ) }
```
Comment (mtias) [inline editor/table-of-contents/index.js]: This could use some better explanation.

**Comment by afercia** [comment]
Moving it away from the content region is a nice improvement thanks 🙂 

This change would also partially solve #3216 because it adds an `aria-label` (and I guess also a tooltip. Maybe the wording `__( 'Information' )` could be improved, but that can be reviewed later, see #2387 
Would be great to add also an `aria-expanded` attribute too.

**Comment by karmatosed** [comment]
Looks great, it's an improvement and opens up the editing area.

**Review by youknowriad — APPROVED** [review]
Nice. 👍

**Comment by mtias** [comment]
@afercia yes, thought you might appreciate it :) Agree on the wording, left a comment about it. Also added expanded role.

**Pull request merged**